modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,60 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+The following guide provides details on removing the Metering Operator and its components.
 
-.Prerequisites
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
+
+== Removing the Metering Operator
+
+You can remove the Metering Operator by following the documentation on xref:../../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+== Uninstalling metering
+
+To uninstall metering, you must remove its custom resource and delete the `openshift-metering` namespace.
 
-* Metering must be installed.
+. Prerequisites
 
-.Procedure
+. You have uninstalled Metering following the documentation listed in the previous step.
 
-To remove metering:
+. Procedure
 
-. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+. Use the following command to remove all resources created by the Metering Operator:
++
+----
+$ oc --namespace openshift-metering delete meteringconfigs.metering.openshift.io openshift-metering
```

Comment:
@timflannagan1 perhaps we can shorten this to `oc delete meteringconfigs <YOUR_METERINGCONFIG_NAME> -n openshift-metering.  Also,

Reply:
Yeah there's some potential variability with the MeteringConfig custom resource name. I think the doc standard is to use something like the following:

```bash
$ oc delete meteringconfig <1> meteringconfigs

<1> The name of the MeteringConfig custom resource present in the openshift-metering namespace.
```